mapie/regression.py

```diff
@@ -275,7 +277,7 @@ def _check_agg_function(
                 "You need to specify an aggregation function when "
                 f"cv's type is in {self.cv_need_agg_function}."
             )
-        if (agg_function is not None) or (self.cv == "prefit"):
+        if (agg_function is not None) or (self.cv in ["prefit", "split"]):
```

Comment:
Define `["prefit", "split"]` as a class attribute

Reply:
I propose to use "self.no_agg_cv_" class attribute.